src/server/server.cpp

```diff
@@ -9,20 +9,53 @@
 
 #include "logs/LogInit.h"
 #include "logs/DefaultLogging.h"
+#include <boost/program_options.hpp>
 
 #include <grpcpp/grpcpp.h>
 #include <grpcpp/health_check_service_interface.h>
 
-int main() {
-    logs_init();
+#include <iostream>
+
+std::string parse_options(int argc, char* argv[]) {
+    bool severity_debug = false;
+    std::string host = "0.0.0.0";
+    std::string port = "8586";
+
+    boost::program_options::options_description desc("Allowed options");
+
+    desc.add_options()
+            ("help", "prints help message")
+            ("debug", boost::program_options::bool_switch(&severity_debug), "enable debug logs")
+            ("host", boost::program_options::value(&host), "server host, default: 0.0.0.0")
+            ("port", boost::program_options::value(&port), "server port, default: 8586");
+
+    boost::program_options::variables_map vm;
+    boost::program_options::store(parse_command_line(argc, argv, desc), vm);
+    boost::program_options::notify(vm);
+
+    if (vm.count("help")) {
+        std::cout << desc << "\n";
+        return "";
+    }
+
+    if (severity_debug)
+        logs_init(boost::log::trivial::severity_level::debug);
```

Comment:
I saw this code already...
Why is it repeating?

Reply:
moved parameter parsing to separate function